src/rules/noTrailingWhitespaceRule.ts

```diff
@@ -41,6 +41,11 @@ export class Rule extends Lint.Rules.AbstractRule {
 }
 
 class NoTrailingWhitespaceWalker extends Lint.SkippableTokenAwareRuleWalker {
+    // prevents skipping of JsxText nodes
+    protected visitJsxText(node: ts.JsxText) {
+        this.walkChildren(node);
```

Comment:
how about adding a constructor to `SkippableTokenAwareRuleWalker` with a parameter `skipJsx`?